### PR TITLE
[QA] CLAUDIA-4315 Open UDP ports and ranges 

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers
@hmunfru 

#### Description
New test cases for CLAUDIA-4314: UDP Ports and Ranges in Sec. Groups.
Refactor due to fixed bugs: Tags and expected results

#### Testing
Locally.

### CODE REVIEW

File: test/acceptance/component/add_product/features/add_product.feature [hunk 1]
```diff
@@ -150,6 +150,9 @@ Feature: Add a new product in the catalogue
     | testing_attributes_231 | product with testing purposes | application/json  | open_ports    | 80 22 8080 8091                              |
     | testing_attributes_232 | product with testing purposes | application/xml   | open_ports    | 80                                           |
     | testing_attributes_233 | product with testing purposes | application/json  | open_ports    |                                              |
+    | testing_attributes_250 | product with testing purposes | application/json  | open_ports    | 80-90                                        |
+    | testing_attributes_251 | product with testing purposes | application/json  | open_ports    | 20 80-90                                     |
+    | testing_attributes_252 | product with testing purposes | application/json  | open_ports    | 20 80-90 65535                               |
```
Comment (flopezag) [inline test/acceptance/component/add_product/features/add_product.feature]: Change it to:

0 80-90 65535

and include the port -20 to test.

File: test/acceptance/e2e/delete_node/features/delete_node.feature [hunk 2]
```diff
@@ -121,10 +121,9 @@ Feature: Remove nodes from chef/puppet servers with E2E validations
...
 
-    @skip @CLAUDIA-4210
     Scenario: Remove non-existent node
 
       Given a configuration management with "<cm_tool>"
       And a node name "nonexistent_node"
       When I remove the node
-      Then I obtain a "400" HTTP error
\ No newline at end of file
+      Then I obtain a "404" HTTP error
```
Comment (flopezag) [inline test/acceptance/e2e/delete_node/features/delete_node.feature]: Introduce newline at the end of the file

Comment (jframos) [inline test/acceptance/e2e/delete_node/features/delete_node.feature]: Added newline.

**Comment by hmunfru** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**